Source/SwiftLintFramework/Extensions/NSFileManager+SwiftLint.swift

```diff
@@ -9,8 +9,8 @@
 import Foundation
 
 extension NSFileManager {
-    internal func filesToLintAtPath(path: String) -> [String] {
-        let absolutePath = path.absolutePathStandardized()
+    internal func filesToLintAtPath(path: String, rootDirectory: String? = nil) -> [String] {
+        let absolutePath = (path.absolutePathRepresentation(rootDirectory ?? NSFileManager.defaultManager().currentDirectoryPath) as NSString).stringByStandardizingPath
```

Comment:
It was both issues. Lines were too long + the above unit test failing to compile. Update coming soon.